### PR TITLE
renaming "messages behind" to "consumer lag"

### LINKED ISSUE #3727: Consumers: rename "messages behind" to "consumer lag"

### PR DESCRIPTION
**Pull request by sukanth**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Closes #3727

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/Topic/ConsumerGroups/TopicConsumerGroups.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const TopicConsumerGroups: React.FC = () => {
         enableSorting: false,
       },
       {
-        header: 'Messages Behind',
+        header: 'consumer lag',
```
Comment (Haarolean) [inline kafka-ui-react-app/src/components/Topics/Topic/ConsumerGroups/TopicConsumerGroups.tsx:51]: Please start it with an upper case letter as well

**Review by Haarolean — CHANGES_REQUESTED** [review]

File: kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalTopicConsumerGroup.java [hunk 2]
```diff
@@ -46,8 +46,8 @@ public static InternalTopicConsumerGroup create(
   }
 
   @Nullable
-  private static Long calculateMessagesBehind(Map<TopicPartition, Long> committedOffsets,
```
[the remark this answers is not in the document]
Comment (sukanth) [reply kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/InternalTopicConsumerGroup.java:49]: should i just leave this file and not update with any naming change ? @Haarolean

File: kafka-ui-react-app/src/components/Topics/Topic/ConsumerGroups/TopicConsumerGroups.tsx (hunk 1 above)
Comment (sukanth) [reply kafka-ui-react-app/src/components/Topics/Topic/ConsumerGroups/TopicConsumerGroups.tsx]: Done

**Review by Haarolean — APPROVED** [review]

**Review by Mgrdich — APPROVED** [review]

**Review by David-DB88 — APPROVED** [review]

**Comment by Haarolean** [comment]
@sukanth thanks for your contribution! :)

**Pull request merged**